aldryn_translation_tools/utils.py

```diff
@@ -57,9 +57,12 @@ def get_object_from_request(model, request,
         # If the model is translatable, and the given slug is a translated
         # field, then find it the Parler way.
         filter_kwargs = {slug_field: kwargs[slug_url_kwarg]}
-        translated_fields = model._parler_meta.get_translated_fields()
+        try:
+            translated_fields = model._parler_meta.get_translated_fields()
+        except AttributeError:
+            translated_fields = None
```

Comment:
you could save a condition by making this an empty list and simply checking if slug_url_kwarg in translated_fields